MAINTAINERS.md

```diff
@@ -14,35 +14,52 @@ small it may be, even a comment on an issue a valuable contribution to us.
 Maintainership requires time commitment and community trust. You don't have to
 be a maintainer to have a meaningful impact on Kubernetes!
 
-## How to become a maintainer
+## Maintainership Requirements
+
+In order to become a maintainer, a contributor must satisfy all of the
+requirements listed below.
 
 ### 1. Be an active contributor
 
-An active contributor is someone who contributed regularly in the last 2 months
-and is one of the top 20 of contributors for the last 2 months.
+Active contributors are considered to be anyone who meets any of the following criteria:
+
+- Sent more than two pull requests (PRs) in the previous one month, or more than
+  20 PRs in the previous year.
+- Filed more than three issues in the previous month, or more than 30 issues in
+  the previous 12 months.
+- Commented on more than pull requests in the previous month, or more than 50
+  pull requests in the previous 12 months.
+- Marked any PR as LGTM in the previous month.
+- Have collaborator permissions in the Kubernetes GitHub organization.
 
-For example in the context of merged commit contributions, from September to
-November, the 20th developer has 20 commits.
+See [Kubernetes Community Expectations](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/community-expectations.md) for more details.
```

Comment:
The requirements of becoming an active contributor are described in [community expectations](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/community-expectations.md), it will probably be moved out of there once we proceed with contributor ladders